modules/3d-tiles/src/tileset/tile-3d-header.js

```diff
@@ -586,6 +586,7 @@ export default class Tile3DHeader {
       this._content = null;
       this.hasEmptyContent = false;
       this.contentState = TILE3D_CONTENT_STATE.UNLOADED;
+      this.fullUri = this._basePath + '/' + this.contentUri;
```

Comment:
I seem to remember that we had a function that did a more sophisticated calculation (perhaps in the base tileset), if so we should probably use it, but I am sure this is good enough for unique hashing.